change/@ni-nimble-components-c5cf9aa2-8b3b-430e-99db-7c820f097578.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@m-akinc fyi I picked up the mapping column while you were out and pivoted direction in some areas: https://github.com/ni/nimble/tree/mraj-mapping-column

Same shifts:
- Mapping elements are now completely standalone / have no knowledge of anything table column specific
- Validation was modified so there is a shared base class for common validation
- attribute / property setting was modified so validation can run prior to connection to the dom since they don't depend on anything visual
- Change to icon validation so that icons can resolve asynchronously
- Started but did not finish label validation
- Columns now create a Map of values to MappingConfig to pass via columnConfig
- Started but haven't finished making the cell view / group header templates less generic, i.e. they don't try and take arbitrary templates but instead support specific MappingConfig objects. This localizes the template logic in those templates.
- Started but haven't finished decoupling the view templates from the text column view templates. Seemed like leveraging those templates was adding unexpected complexity as features weren't always needed like placeholder
- Discussed with team and aligned on removing support for default-mapping and placeholder. Started but have not finished that change. Will likely remove placeholder from other column types as well, but need to capture that discussion in an issue
- Started but haven't finished a tweak to the validator pattern to make typing easier in the future (may be a follow-up instead)

Just wanted to let you know the state and the areas I plan on continuing to look at post quiet week.